lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1430,32 +1429,12 @@ getNetworkInformation (_block0, gbp, st) nl = do
 
 getNetworkParameters
     :: (Block, GenesisBlockParameters, SyncTolerance)
-    -> ApiEpochNumber
     -> Handler ApiNetworkParameters
-getNetworkParameters (_block0, gbp, _st) apiEpochNum = do
-    case apiEpochNum of
-        ApiEpochNumber epochNum -> do
-            now <- liftIO getCurrentTime
-            let slotParams = W.slotParams bp
-            let ntrkTip = fromMaybe slotMinBound (slotAt slotParams now)
-            let currentEpochNum = ntrkTip ^. #epochNumber
-            when (currentEpochNum < epochNum) $
-                liftHandler $ throwE $ ErrNoSuchEpoch
-                    { errGivenEpoch = epochNum
-                    , errCurrentEpoch = currentEpochNum
-                    }
-            pure (toApiNetworkParameters bp)
-
-        ApiEpochNumberLatest ->
-            pure (toApiNetworkParameters bp)
+getNetworkParameters (_block0, gbp, _st) =
+    pure $ toApiNetworkParameters bp
```

Comment:
Consider removing the where clause and putting it all on one line.

Reply:
Good point! I've added this to the list of things to do on the next PR: https://github.com/input-output-hk/cardano-wallet/issues/1692